velox/exec/tests/utils/DistributedPlanBuilder.cpp

```diff
@@ -108,6 +108,29 @@ core::PlanNodePtr DistributedPlanBuilder::shuffleResult(
   return std::move(planNode_);
 }
 
+core::PlanNodePtr DistributedPlanBuilder::broadcastResult() {
```

Comment:
Shall we call this shuffleBroadcastResult? And rename the existing one to shufflePartitionedResult?